spec/services/digest_initiator_service_spec.rb

```diff
@@ -9,31 +9,33 @@
 
     context "when a digest run isn't processed" do
```

Comment:
If you don't mind going a little bit further this file could benefit from a frozen time as I believe most of the tests in this fail before 8am https://ci.integration.publishing.service.gov.uk/job/email-alert-api/job/dependabot%252Fbundler%252Frails-6.0.3.3/1/console

Reply:
Sure, I've added another commit to do that for this class and the DigestRun model specs.